### DISCUSSION TITLE
Facecam for FCFP replay mode

### DISCUSSION THREAD

**Discussion by naminode** [kinsi55/CS_BeatSaber_Camera2#66] [Q&A] [answered]
Hi . The current facecam does not work with fcfp mode as shown in the pic below. Is it possible to have a fcfp version of the facecam? 

![image](https://user-images.githubusercontent.com/88361811/156937006-ffa3d5ab-f73e-4097-9cda-0f71c6653b9d.png)

**Comment by kinsi55** [comment, marked as answer]
Enable "Follow replay position" under Smoothing for the Facecam

This is off by default so that the Camera would still follow your real head if you were to watch a replay from VR

**Reply by naminode** [reply]
Have tried to adjust the fov and max out z offset .Also avatar forced visuble in FP . Still unable to get the avatar into camera.
![image](https://user-images.githubusercontent.com/88361811/156937818-f277493a-42b2-48d8-8d68-2fbfbb46bd15.png)

**Reply by kinsi55** [reply]
Don't touch any option other than the follow replay pos. In doubt recreate the cam
